packages/flutter_tools/templates/package_ffi/go/lib/projectName.dart.tmpl

```diff
@@ -0,0 +1,109 @@
+import 'dart:async';
+import 'dart:isolate';
+
+import 'bridge.obj.dart' ;
+
+
+/// A very short-lived native function.
+///
+/// For very short-lived functions, it is fine to call them on the main isolate.
+/// They will block the Dart execution while running the native function, so
+/// only do this for native functions which are guaranteed to be short-lived.
+int sum(int a, int b) => getBridge().sum(a, b);
+
+/// A longer lived native function, which occupies the thread calling it.
+///
+/// Do not call these kind of native functions in the main isolate. They will
+/// block Dart execution. This will cause dropped frames in Flutter applications.
+/// Instead, call these native functions on a separate isolate.
+///
+/// Modify this to suit your own use case. Example use cases:
+///
+/// 1. Reuse a single isolate for various different kinds of requests.
+/// 2. Use multiple helper isolates for parallel execution.
+Future<int> sumAsync(int a, int b) async {
```

Comment:
Any reason for not using the autogenerated `sumLongRunningAsync` that uses ports behind the scenes?